spec/system/planning_applications/permitted_development_right_spec.rb

```diff
@@ -361,6 +361,61 @@
         expect(page).to have_content("When were the works completed? 01/02/2015")
         expect(page).to have_content("Has anyone ever attempted to conceal the changes? No")
         expect(page).to have_content("Has enforcement action been taken about these changes? No")
+
+        expect(page).to have_content("Assessment summary — Evidence of immunity")
+      end
+
+      context "when it has no evidence attached" do
```

Comment:
possibly worth copying all these specs over to `review_immunity_detail_permitted_development_right_spec.rb`